### DISCUSSION TITLE
🎉 Version: 2.0.0

### DISCUSSION THREAD

**Discussion by mertushka** [mertushka/haxball.js#4] [Announcements]
# What's Changed
- Removed puppeteer module and switched it with native Haxball Headless API.
- Update README.md
---
***Basicly, everything is changed.***

<hr /><em>This discussion was created from the release <a href='https://github.com/mertushka/haxball.js/releases/tag/2.0.0'>🎉 Version: 2.0.0</a>.</em>

**Comment by ImmersHaxball** [comment]
Great update dude, thanks a lot for your work! :)

**Reply by mertushka** [reply]
Thanks, you're welcome! 👍
